docs/source/conf.py

```diff
@@ -46,11 +46,11 @@
 ]
 
 intersphinx_mapping = {
-    "python": ("https://docs.python.org/3.6", None),
-    "sklearn": ("http://scikit-learn.org/stable/", None),
+    "python": ("https://docs.python.org/3.10", None),
```

Comment:
I _think_ this should be slightly more future proof by pointing to whatever the latest Python release is 

```suggestion
    "python": ("https://docs.python.org", None),
```